monitoring-custom-metrics/README.md

```diff
@@ -2,189 +2,51 @@
-
-
-- The reference documentation of the [package prometheus](https://godoc.org/github.com/prometheus/client_golang/prometheus).
-
-- Official prometheus documentation of [metric types](https://prometheus.io/docs/concepts/metric_types/).
-
-#### Access the exposed mentrics in Prometheus
-
-Run the port-forward on prometheus-core-0 which is the Prometheus server running in Kyma.
-```bash
-$ kubectl port-forward pod/prometheus-core-0 -n kyma-system 9090:9090
-Forwarding from 127.0.0.1:9090 -> 9090
-Forwarding from [::1]:9090 -> 9090
-```
-You will see in **target** all the **sample-metrics** endpoint. **sample-metrics-8081** is the name of the Service discovered by the **ServiceMonitor**
-
-![Prometheus Dashboard](images/pm-dashboard-1.png)
-
-
-#### ServiceMonitor
-
-Prometheus is able to reach the service thanks to the ServiceMonitor which has been created together with the sample metrics application. A ServiceMonitor is a specific CRD used by the prometheus operator to monitor services.
-
-In Kyma the prometheus server discovers all ServiceMonitor through a ```serviceMonitorSelector:``` matching the label ```prometheus: core```
-
-```yaml
-  serviceMonitorSelector:
-    matchLabels:
-      prometheus: {{ .Values.prometheusLabelValue | default .Release.Name | quote }}
-{{- end }}
-```
-
-
-In our example and at the same way, the ServiceMonitor selects, ```selector:```, all Service matching a label, ```k8s-app: metrics``` and target the label.
-
-- [Service and ServiceMonitor](k8s/sample-metrics-app.yaml)
-
-In Kyma we have a template, [kyma/resources/core/charts/monitoring/charts/prometheus/templates/servicemonitors.yaml](https://github.com/kyma-project/kyma/blob/master/resources/core/charts/monitoring/charts/prometheus/templates/servicemonitors.yaml), which is ```kind: List``` to discover a list of ServiceMonitors.
-
-### Adding Custom Dashboard in Grafana
-
-As mentioned, Kyma includes a set of dashboards. However, users can create their own **Grafana Dashboard** by using the Grafana UI as the dashboards are persisted even after the pod restarts.
-
-For details of how to create dashboards in Grafana:
-- [Grafana in Kyma](https://github.com/kyma-project/kyma/blob/master/resources/core/charts/monitoring/charts/grafana/README.md)
-- [Grafana - Getting started](http://docs.grafana.org/guides/getting_started/)
-- [Export and Import dashboards](http://docs.grafana.org/reference/export_import/)
-- [Grafana - Dashboard API](http://docs.grafana.org/http_api/dashboard/)
-
-### Cleanup
-Run the following commands to completely remove the example and all its resources from the cluster:
-
-- Remove label istio-injection
-  ```$ kubectl label namespace default istio-injection-```
-- Remove ServiceMonitor in namespace kyma-system
-  ```$ kubectl delete servicemonitor -l example=monitoring-custom-metrics -n kyma-system```
 - Remove sample-metrics deployments in namespace default

```

Comment:
``Remove the `sample-metrics` Deployments in the `default` Namespace.``

Reply:
done